app/assets/stylesheets/theme-geniventure.scss

```diff
@@ -1,92 +1,55 @@
 @import "cc-runtime";
 
-$src: url('/assets/geniventure-bg/mission-control.jpg');
-$c_a: #f88c22;
-$c_b: #f5a744;
-
-@include theme($src, $c_a, $c_b);
-
-$dark-olive-green: #402e17;
-$dark-khaki: #b0925c;
-$dim-gray: #4c4a42;
-$sea-green: #a7be99;
-$dark-sea-green: #7d8d75;
-
-header {
-  margin-bottom: 0;
-}
-
-.site-logo.logo-l {
-  margin-left: 6px;
-  img {
-    max-height: 70px;
-  }
-}
-.site-logo.logo-r {
-  margin-right: 9px;
-  img {
-    margin-top: 14px;
-  }
-}
+$col-dark-olive-green: #402e17;
+$col-dark-khaki: #b0925c;
+$col-dim-gray: #4c4a42;
+$col-sea-green: #a7be99;
+$col-dark-sea-green: #7d8d75;
+$col-orange: #f88c22;
+
+$theme-params: (
+  bgColorOrUrl: url('/assets/geniventure-bg/mission-control.jpg'),
+  theme-color-a: $col-dark-olive-green,
+  theme-color-b: $col-orange,
+  activity-menu-display: none
```

Comment:
Have you checked if this activity-menu-display option is working in this case?
My guess is that the 'splat' approach of `@include theme($theme-params...);` just takes the theme-params turns it into an array and then uses each value as a parameter. So my guess is it the activity-menu-display ends up in the right-logo-display parameter.

Reply:
Yeah, I checked. I've just double-checked to be sure. It works correctly.

As I understand it, and as it appears to function, so long as you specify the right parameter name, the mixin will assign the corresponding value to that parameter. The order doesn't matter and you can leave out these new parameters without it breaking things. It seems pretty flexible and I thought it'd be a good method to use so you don't have to follow a set order and include values for things you're not concerned with, especially if we keep adding more parameters. Does that make sense?